bootstrap/023_simple_materializations.sql

```diff
@@ -128,9 +128,9 @@ begin
         {
           'warehouse_name': :warehouse_name,
           'start_time': dateadd(hours, value, :start_time)::text,
-          'end_time': dateadd(hours, value+8, :start_time)::text
+          'end_time': dateadd(hours, value+7, :start_time)::text
         })
-    from table(flatten(input=>array_generate_range(0, datediff(hours, :start_time, current_timestamp()), 8))) f);
+    from table(flatten(input=>array_generate_range(0, datediff(hours, :start_time, current_timestamp()), 7))) f);
```

Comment:
if the logic worked before, it doesn't seem like this needs to change. we only need to change the schedule, no?

Also, I think you need to update the task log summary view.

Reply:
> if the logic worked before, it doesn't seem like this needs to change. we only need to change the schedule, no?

I was waffling on it, honestly. Happy to walk this back.

> Also, I think you need to update the task log summary view.

Ah, and now I know what you meant! :)